components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/OAuthInfo.kt

```diff
@@ -31,9 +31,9 @@ class OAuthInfo internal constructor(raw: Raw) {
     }
 
     init {
-        this.accessToken = raw.access_token
-        this.keys = raw.keys
-        this.scope = raw.scope
+        this.accessToken = if (raw.access_token == null) null else raw.access_token!!.getString(0, "utf8")
```

Comment:
I believe I got it from an example in the JNA repo. Also, this indicates that it would like it spelt this way https://java-native-access.github.io/jna/4.2.1/constant-values.html#com.sun.jna.Native.DEFAULT_ENCODING.

(Note that AFAICT the version of getString that doesn't specify the encoding can be overridden by a system property, which is bad because the Rust strings are *always* utf-8)

Reply:
@thomcc nit: Could we then use the Native.DEFAULT_ENCODING constant instead of repeating the "utf8" value?

Reply:
Sure, although I'm not sure I think that's an improvement. Rust is documented as using UTF-8 as it's string encoding. AFAICT the fact that JNA also uses UTF-8 for DEFAULT_ENCODING is something of a coincidence. If a future update changed it (as unlikely as that sounds), the code would break in that case.

Reply:
OK, if we want to guard against changes here, we can also create our own ENCODING constant, which still seems better to me than copying the string many times :).

Reply:
I've opted to go with that. I've also moved the existing uses of getString to use the new constant.